backend/common/models/auth.py

```diff
@@ -0,0 +1,6 @@
+from django.contrib.auth.models import AbstractUser
+
+
+class User(AbstractUser):
+    USERNAME_FIELD = "email"
+    pass
```

Comment:
Jakmile ma trida aspon jeden radek nejakeho smysluplneho kodu, `pass` uz neni treba.

Reply:
Model pro uzivatele bych asi vyclenil do samostatne aplikace `profile` (je to jasne ohranicena domena). Do common bych daval opravdu jen obecnosti.